providers/vmware_simulator.md

```diff
@@ -0,0 +1,98 @@
+# installing vmware simulator
+
+```bash
+cd ~/go/
+git clone https://github.com/kbrock/govmomi
+cd govmomi
+git remote add upstream  https://github.com/vmware/govmomi
+git checkout miq_counters_kb
+# client
+pushd govc ; make clean && make && make install ; popd
+# server
+pushd vcsim ; make clean && make && make install ; popd
+ln -s ~/go/bin/vcsim ~/bin # or /usr/local/bin
+ln -s ~/go/bin/govc ~/bin  # or /usr/local/bin
+```
+
+# creating simulator provider
+
+```bash
+echo "127.0.0.1 vc91" > /etc/hosts
+
+# run temporary simulator 
+vcsim -username root -password vmware -app 2 -cluster 2 -dc 1 -ds 2 -folder 2 \
+      -host 2 -nsx 2 -pg 2 -pg-nsx 2 -pod 2 -pool 1 -standalone-host 2 -vm 2 \
+      -l vc91:9191 &
```

Comment:
Not required, but I tend to find guides more readable if they are one param per line.  It also allows easier modification for things like removing lines.

```suggestion
vcsim \
   -username root \
   -password vmware \
   -app 2 \
   -cluster 2 \
   -dc 1 \
   -ds 2 \
   -folder 2 \
   -host 2 \
   -nsx 2 \
   -pg 2 \
   -pg-nsx 2 \
   -pod 2 \
   -pool 1 \
   -standalone-host 2 \
   -vm 2 \
   -l vc91:9191 &
```

Also is that `-|` at the end there  intentional?